packages/actor-http-native/lib/Requester.ts

```diff
@@ -38,8 +38,8 @@ export default class Requester {
     // Unpacking headers object into a plain object
     const headersObject: any = {};
     if (settings.headers) {
-      for (const header of settings.headers) {
```

Comment:
This change should not be needed.
If this is not iterable, then it means that `settings.headers` is invalid, and something is going wrong elsewhere.